blocks/api/serializer.js

```diff
@@ -166,11 +166,11 @@ export function serializeBlock( block ) {
 }
 
 /**
- * Takes a block list and returns the serialized post content.
+ * Takes a block or set of blocks and returns the serialized post content.
  *
- * @param  {Array}  blocks Block list
+ * @param  {Array}  blocks Block(s) to serialize
  * @return {String}        The post content
  */
 export default function serialize( blocks ) {
-	return blocks.map( serializeBlock ).join( '\n\n' );
+	return castArray( blocks ).map( serializeBlock ).join( '\n\n' );
```

Comment:
>Curious, why is this needed?

It's not strictly necessary. In this case the requirement was: being able to serialize a single block. I could have achieved this without any changes to `serialize` by wrapping in an array i.e. `serialize( [ block ] )`, but it occurred to me there's no real reason `serialize` shouldn't be able to be called with a single argument, so this merely normalizes the value.

An alternative approach could also have skipped `map` altogether for singular argument and just `return`ed `serializeBlock`.

Yet another alternative is to expose the individual `serializeBlock` on the block API, and use the specific method appropriate to single / array forms.